proknow-sdk-test/PatientTest/EntitiesTest/EntityScorecardItemTest.cs

```diff
@@ -145,6 +145,7 @@ public async Task SaveAsyncTest()
                 new MetricBin("FAIL", new byte[] { 255, 0, 0 })
             };
             entityScorecardItem.CustomMetrics = new List<CustomMetricItem>() { customMetricItem2 };
+            //TODO--Modify custom metric value and verify that it is saved
```

Comment:
Did you mean to come back to this before submitting this PR or is this something for the future?

Reply:
This is something for the future.